docs/source/integrations/labelbox.rst

```diff
@@ -687,7 +687,7 @@ take additional values:
 Note that only scalar-valued label attributes are supported. Other attribute
 types like lists, dictionaries, and arrays will be omitted.
 
-.. note:: 
+.. note::
```

Comment:
nit: please tweak your editor so that it only removes trailing whitespace on lines that you edit, not the entire file.

Looks like yours is configured to process the whole file, which can lead to large cosmetic diffs whenever you touch a file.